serviceWorker.js

```diff
@@ -0,0 +1,98 @@
+// Set to false to disable logging.
+const verbose = true;
+
+async function handleFetch(request) {
+	// If the request is cache-only and the mode is not same-origin, return nothing
+	if (request.cache === "only-if-cached" && request.mode !== "same-origin") {
```

Comment:
Can you elaborate about the purpose of this check? Do we even use "cache-only" requests in webvm?

Reply:
Seems to have been a work-around for an issue on chromium, that seems to have been fixed recently. I am not able to find occurrences of cache-only in Webvm. 

I  will remove it.